modules/savanna/modules/workgroup-workspace/pages/workspaces/workspace.adoc

```diff
@@ -25,6 +25,38 @@ It holds the persistent data that is loaded into TigerGraph and is accessible fo
 We only support a maximum of one Read-Write workspace to connect to a database.
 ====
 
+
+=== Operations Restricted in a Read-Only (RO) Workspace
+
+In a Read-Only (RO) workspace within TigerGraph Savanna, the following operations are restricted:
+
```

Comment:
Saying something "is restricted" means there are some limitations. It does not specify the limitations, and it does not say it is prohibited. For example, think of a sign on a gate that says, "Permission to enter is restricted."

"In RO workspaces, database operations are restricted": true, but by itself, doesn't say enough to be useful.
"In RO workspaces, DROP ALL is restricted": Wrong word. Better: "In RO workspaces, DROP ALL is prohibited."

I suggest first added a better intro sentence, and then using "prohibited" instead of "restricted":

As the name implies, Read-Only workspaces only allow users to read the data in the graph, not to modify.
Accordingly, the following operations are prohibited: